### ISSUE TITLE
Who is the real developer?

### ISSUE THREAD

**Issue by Anutrix** [ximaera/vulners-browser-scanner#1]
Is it https://github.com/vankyver or https://github.com/ximaera? We were thinking of similar add-on based on this(since this seems dead). We would like more info regarding the project. Building guide would also be helpful.

**Comment by ximaera** [comment]
Hi! Are you interested in a Chrome extension or a Firefox extension? Those were developed in two different branches.

**Comment by Anutrix** [comment]
Mainly Firefox. But since both Chrome and Firefox use WebExtension API so aiming for a cross-platform approach.

**Comment by ximaera** [comment]
Yep. We've run into some difficulties with the cross-platform approach 2 years ago ;-)

- Here's how you run it for yourself: https://github.com/ximaera/vulners-browser-scanner/blob/firefox/README.MD
- Here's how you can pack it to distribute: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_first_WebExtension

The extension is based on Vulners API: https://github.com/ximaera/vulners-browser-scanner/blob/firefox/background.js

API documentation: https://vulners.com/api/v3/

**Comment by Anutrix** [comment]
> 
> 
> Yep. We've run into some difficulties with the cross-platform approach 2 years ago ;-)
> 
>     * Here's how you run it for yourself: https://github.com/ximaera/vulners-browser-scanner/blob/firefox/README.MD
> 
>     * Here's how you can pack it to distribute: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_first_WebExtension
> 
> 
> The extension is based on Vulners API: https://github.com/ximaera/vulners-browser-scanner/blob/firefox/background.js
> 
> API documentation: https://vulners.com/api/v3/

Thx a lot for the info and the timers. Also, if you can, do mention the difficulties you faced so that we can think of something. We were thinking of making combination of this and Wappalyzer.
